prime-angular-frontend/src/app/modules/phsa-labtech/phsa-labtech.routes.ts

```diff
@@ -1,5 +1,5 @@
 export class PhsaLabtechRoutes {
-  public static PHSA_LABTECH = 'phsa';
+  public static PHSA_LABTECH = 'phsa-labtech';
```

Comment:
Infinite loop created when routing to the default route of PHSA and not being able to distinguish between it and the AuthModule route which were both `/phsa`

Reply:
Is this a problem or not?  User bookmarks `http://.../phsa-labtech/demographic`, logs out, and later returns to bookmark.  Then they are directed to `http://.../info` ... probably not what they expected?  Simply User Error?

Reply:
Suspect that it is a user that submits, remains logged in, receives an email with a link to login, but since they were already logged in they hit the redirect guard and go to sites.  The infinite loop was behind that issue where Angular was having issues due to the `http://localhost:4200/phsa` route being the same for PHSA login and PhsaLabtechModule.